Quadriga/src/main/java/edu/asu/spring/quadriga/service/network/mapper/impl/NetworkMapper.java

```diff
@@ -53,6 +53,7 @@ public INetwork getNetwork(String networkId) throws QuadrigaStorageException
 			network = networkFactory.createNetworkObject();
 			network.setNetworkId(networksDTO.getNetworkid());
 			network.setNetworkName(networksDTO.getNetworkname());
+		//	network.setCreatedDate(networkDTO.getCreateddate());

```

Comment:
This was a comment added by me and was not originally present as part of code. I have removed it